R/theme-elements.r

```diff
@@ -120,6 +121,37 @@ element_text <- function(family = NULL, face = NULL, colour = NULL,
   )
 }
 
+#' @param colour.accent1,color.accent1 accent colour 1,
+#'   typically a lighter version of colour
+#' @param colour.accent2,color.accent2 accent colour 2,
+#'   typically a bright colour used for geom_smooth et al.
+#' @param fill.accent accent fill colour, typically a darker version of fill
+#' @param alpha colour/fill transparency, between 0 & 1.
```

Comment:
Do we need alpha as a default? People could always set individually with `alpha()`

Reply:
Presumably not. Currently `alpha` has to be NA by default to respect RGBA colour specification